internal/rest/rpc-stats.go

```diff
@@ -71,8 +69,6 @@ func setupReqStatsUpdate(req *http.Request) (*http.Request, func()) {
 		ConnectDone: func(network, addr string, err error) {
 			if err == nil {
 				atomic.StoreInt64(&dialEnd, time.Now().UnixNano())
-			} else {
-				logger.LogOnceIf(req.Context(), logSubsys, err, req.URL.Hostname())
```

Comment:
Not sure if this is intentional, because it doesn't look like it's related to this PR.

Reply:
yes its intentional, this logging was super spurious. Just added it in this PR anyway a Bonus change.